src/display.c

```diff
@@ -74,24 +83,44 @@ static void OnDestroySurfaceRecord(void *data)
 	SurfaceRecord record = data;
 	Surface_Close(record->surface);
 	LinkedList_Clear(&record->rects, free);
+	LinkedList_Clear(&record->flashRects, free);
 	free(record);
 }
 
-static void DrawBorder(LCUI_PaintContext paint)
+static void UpdateFlashRects(SurfaceRecord record)
 {
-	LCUI_Pos pos;
-	LCUI_Color color;
-	int end_x = paint->rect.width - 1;
-	int end_y = paint->rect.height - 1;
-	pos.x = pos.y = 0;
-	color = RGB(255, 0, 0);
-	Graph_DrawHorizLine(&paint->canvas, color, 1, pos, end_x);
-	Graph_DrawVertiLine(&paint->canvas, color, 1, pos, end_y);
-	pos.x = paint->rect.width - 1;
-	Graph_DrawVertiLine(&paint->canvas, color, 1, pos, end_y);
-	pos.x = 0;
-	pos.y = paint->rect.height - 1;
-	Graph_DrawHorizLine(&paint->canvas, color, 1, pos, end_x);
+	LinkedListNode *fr;
```

Comment:
Variable of type `LinkedListNode` should be preferentially named `node`, especially if there is only one object of type `LinkedListNode` in the current scope.